src/internal/connector/sharepoint/collection.go

```diff
@@ -0,0 +1,55 @@
+package sharepoint
+
+import (
+	"io"
+
+	"github.com/alcionai/corso/src/internal/connector/graph"
+	"github.com/alcionai/corso/src/internal/connector/support"
+	"github.com/alcionai/corso/src/internal/data"
+	"github.com/alcionai/corso/src/pkg/path"
+)
+
+type Target int
```

Comment:
`sharepoint.Target` doesn't seem like the correct description. Should this be `DataCategory` or `CollectionType` ?

Reply:
Resolved with [d25487b](https://github.com/alcionai/corso/pull/1444/commits/d25487b636f93de417320fb8bed83cdc47c068c9)